typescript/src/main.ts

```diff
@@ -94,6 +112,33 @@ function appendMetalTableRow(metal: metals.Metal, table: HTMLTableElement) {
   resultCell.classList.add("result");
   resultCell.id = "result_" + metal.idSuffix;
 }
+function toggleMetal(bool: boolean, metal: metals.Metal) {
```

Comment:
Sorry, one more thing: if you give the bool a name like `isOff` it's easier to see later on which way round things should work.